ellie/monopoly.py

```diff
@@ -0,0 +1,154 @@
+from random import randint
+
+class Player:
+    def __init__(self, player_name, player_token):
+        self.name = player_name
+        self.token = player_token
+        self.money = 1500
+        self.properties = []
+        self.position = 0
+        self.in_jail = False
+        self.is_turn = False
+    
+    def __repr__(self):
+        return f"This player, {self.name}, is playing as the {self.token}. \n{self.name} has ${self.money}, is currently on {board[self.position]}, and owns these properties: {self.properties}"
+
+    def roll_dice(self):
+        # Roll two dice and return the results
```

Comment:
In Python, there's a concept called [docstrings](https://peps.python.org/pep-0257/) where the first line of a method/class/module/etc is actually a string literal. On a technical level, this makes it available as the `__doc__` attribute on the thing you're documenting (e.g. `Player.roll_dice.__doc__`), but also there are many tools for Python that use these strings to generate documentation.

Docstrings commonly (but not _necessarily_) use triple-quote strings since they allow for easy multi-line docstrings if needed:

```py
def roll_dice(self):
    '''Roll dice and return the results.'''
    die1 = randint(1, 6)
```

In general it's a good habit to follow the convention of using docstrings when documenting methods and such.